### PR TITLE
ci: Allow to run via docker despite using dependencies

### PR DESCRIPTION
**Pull request by jens1o**

### CODE REVIEW

File: README.md
Comment (Jonas-Karl) [inline README.md:1]: Thanks for updating the readme!

**Comment by Jonas-Karl** [comment]
Works perfectly. Do you know why the System.err is now also white? Would be a better ux if there was color

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,14 @@
 #
 # Build stage
 #
-FROM maven:3.9.5-eclipse-temurin-17-focal AS build
+FROM maven:3.9.6-eclipse-temurin-17-focal AS build
 COPY src /home/app/src
 COPY pom.xml /home/app
-RUN mvn -f /home/app/pom.xml clean package
+RUN mvn -f /home/app/pom.xml package
+
```
Comment (Jonas-Karl) [inline Dockerfile:8]: Is it correct that there is no clean build anymore?

Comment (jens1o) [reply Dockerfile:8]: well as far as I see it made no difference because we only copied the source code, I'll add it back, it does not hurt...

**Review by Jonas-Karl — APPROVED** [review]
Good job!

**Pull request merged**